test/grpc/route_guide/route-guide-utilities.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
```

Comment:
I want to copy what is in https://github.com/grpc/grpc/tree/master/examples/cpp/route_guide as closely as possible (everything but eventuals). Please update accordingly and remove this utilities file, thanks!